src/WordTokenizers.jl

```diff
@@ -17,7 +17,7 @@ export poormans_tokenize, punctuation_space_tokenize,
        set_tokenizer, set_sentence_splitter,
        rev_tokenize, rev_detokenize,
        toktok_tokenize
-export ALBERT_V1, ALBERT_V2, load, tokenizer, sentence_from_tokens, ids_from_tokens
+export ALBERT_V1, ALBERT_V2, load, tokenizer, sentence_from_tokens, ids_from_tokens, GPT2, GPT2Tokenizer, tokenize, sentence_from_tokens_gpt2
```

Comment:
are all these exports needed? In particular I'm afraid that the name `GPT2` here will clash with the actual model whenever implemented.

Reply:
Yeah you're right. There might be a better alternative for this. It is consistent with ALBERT_V1 and goes with load(ALBERT_v1) so i did this. I just realised there's no need to export GPT2Tokenizer as well. I'll correct it.

Reply:
It will be better to have common APIs  for all the statical Tokenizers
For instance, sentence_from_tokens can be shared between ALBERT and GPT2.

Reply:
@tejasvaidhyadev Can I make all the API as of the format `f(tokens/text, tokenizer(spm/gpt2))` instead of `f(tokenizer(spm/gpt2), tokens/text)`? I feel it might be more intuitive for users this way.